### PR TITLE
Add LearnedSort 2.0 pos

### PR DESCRIPTION
**Pull request by anikristo**

### CODE REVIEW

File: _posts/2021-06-26-defeating-dups-learned-sort.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Defeating duplicates: A re-design of the LearnedSort algorithm"
```
Comment (andreaskipf) [inline _posts/2021-06-26-defeating-dups-learned-sort.md:3]: Optional: Use title case to be consistent with the other posts.

File: _posts/2021-06-26-defeating-dups-learned-sort.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Instead, what worked best was to think of buckets as a collection of smaller, fixed-capacity fragments. The eCDF prediction step would be the same; however, instead of placing the key directly to its predicted bucket, we place it to the corresponding pre-allocated fragment owned by the predicted bucket.
+
+{:refdef: style="text-align: center;"}
+![step1](/assets/learnedsort/step1.png){: style="max-width: 36em" }
+{: refdef}
+
+Once a fragment reaches its full capacity (i.e., 100 elements), it is copied back to the input at a given write-head. The write-head is updated, and the copied fragment is cleared to make space for more incoming elements. This procedure continues until the entire input has been processed.
```
Comment (andreaskipf) [inline _posts/2021-06-26-defeating-dups-learned-sort.md:54]: I don't understand this part. How does this terminate?

**Review by andreaskipf — APPROVED** [review]
LGTM. Cool work. Let me know once it's ready to be published. I'll then merge it and announce it.

**Comment by anikristo** [comment]
Hi @andreaskipf, Thanks for your review! This was just meant to be a draft pull request because I still need to do some polishing and add some links. I will let you know once it’s ready to be published. Thank you!

**Comment by anikristo** [comment]
@andreaskipf This post is ready for publishing.

**Pull request merged**